test/templates/action-bar/component.njk

```diff
@@ -1,32 +1,27 @@
-<!--
-action_bar:
-    properties={},
-    modifier={ block: [] },
-    state={},
-    aria={}
--->
-{%- macro action_bar(properties={}, modifier={}, state={}, aria={}) %}
-<div class="fd-action-bar{{ modifier.block | modifier('action-bar') }}">
-    {%- if properties.back_button %}
-    <div class="fd-action-bar__back">
-        <button class="fd-button--light sap-icon--nav-back"></button>
-    </div>
-    {%- endif %}
+{% set headlines = ['h2', 'h3', 'h4', 'h5', 'h6'] %}
+
+{%- macro action_bar(properties={}, aria={}, title_size="h3") %}
+<div class="fd-action-bar">
+{%- if properties.back_button %}
+    <div class="fd-action-bar__back"><div class="test-btn-back sap-icon--nav-back"></div></div>
```

Comment:
Can we replace `test-btn-back` with `btn-playground`?  Also, we should not be using the class `sap-icon--nav-back` in the playground examples.